src/translations/en.json

```diff
@@ -59,6 +59,7 @@
   "cvaa": {
     "title": "Advanced captions settings",
     "sample_caption_tag": "Sample",
+    "sample_custom_caption_tag": "Custom captions",
```

Comment:
I think you need to keep the format of - "first_second": "First second", I mean to remove the sample

Reply:
The reason i left the "sample" was that this is also a sample that shows how the captions look like.